nextcord/embeds.py

```diff
@@ -616,13 +616,21 @@ def insert_field_at(self: E, index: int, *, name: Any, value: Any, inline: bool
 
         return self
 
-    def clear_fields(self) -> None:
-        """Removes all fields from this embed."""
+    def clear_fields(self: E) -> E:
+        """Removes all fields from this embed.
+
+        This function returns the class instance to allow for fluent-style
+        chaining.
+
+        .. versionchanged:: 2.0
+            This method returns ``self``
```

Comment:
I actually might change my mind about the versionchanged. Since it's not a breaking change, I don't know if it's necessary.
```suggestion
```